src/actions/CommandAction.ts

```diff
@@ -67,24 +69,45 @@ export default class CommandAction implements Action {
             await sleep(args.preCommandSleep);
           }
 
-          const [updatedOriginalSelections, updatedTargetSelections] =
-            await callFunctionAndUpdateSelections(
-              this.graph.rangeUpdater,
-              () => commands.executeCommand(args.command!, args.commandArgs),
-              editor.document,
-              [originalSelections, targetSelections]
-            );
+          if (args.awaitCommand) {
+            const [updatedOriginalSelections, updatedTargetSelections] =
+              await callFunctionAndUpdateSelections(
+                this.graph.rangeUpdater,
+                () => commands.executeCommand(args.command!, args.commandArgs),
```

Comment:
Just noticed:

- Command should never be undefined, right?  We should make it required if so so we can avoid the `!`
- We should spread `commandArgs` because some commands take more than one arg.  If this will break anything we should fix that too

```suggestion
                () => commands.executeCommand(args.command, ...args.commandArgs),
```

Reply:
* It is. All arguments are available through the constructor or the run method. In that case they have to be optional. 
* Spreading the command argument creates a compilation error. The current implementation works fine with multiple arguments.